PluginShowcase/nuget.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="AvaloniaCI" value="https://www.myget.org/F/avalonia-ci/api/v2" />
```

Comment:
This is an odd thing to add.

Reply:
That is... odd... I'm going to remove it and make sure stuff still works.

Reply:
Restore worked fine. Removed.